scavenger-api/src/main/kotlin/com/navercorp/scavenger/controller/SnapshotController.kt

```diff
@@ -105,31 +97,12 @@ class SnapshotController(
         )
     }
 
-    @GetMapping("/customers/{customerId}/snapshot/{snapshotId}/export", produces = ["text/csv"])
+    @GetMapping("/customers/{customerId}/snapshots/{snapshotId}/export")
```

Comment:
Should we also rename the methods across the board?
It seems like it's now an API to get all snapshot nodes rather than export.